backend/docker-compose-bare-metal.yaml

```diff
@@ -0,0 +1,196 @@
+
+x-graph-service-env: &graph-service-env
+  QUEUE_HIGH_WATER: 1000
+  DEBOUNCE_SECONDS: 10
+  CAPACITY_LIMIT: '{"type":"percentage", "value":80}'
+  GRAPH_ENVIRONMENT_TYPE: Mainnet
+  PROVIDER_ACCOUNT_SEED_PHRASE: '//Alice'
+  RECONNECTION_SERVICE_REQUIRED: false
+  PROVIDER_BASE_URL: 'http://host.docker.internal:3002/webhooks/graph-service'
+
+x-account-service-env: &account-service-env
+  FREQUENCY_HTTP_URL: 'http://localhost:9944'
+  PROVIDER_BASE_URL: 'http://host.docker.internal:3002/webhooks/account-service'
+
+services:
+  redis:
+    image: redis:latest
+    ports:
+      - 6379:6379
+    networks:
+      - social-app-backend
+    volumes:
+      - redis_data:/data/redis
+
+  frequency:
+    image: dsnp/instant-seal-node-with-deployed-schemas:latest
+    # We need to specify the platform because it's the only image
+    # built by Frequency at the moment, and auto-pull won't work otherwise
+    platform: linux/amd64
+    # Uncomment SEALING_MODE and SEALING_INTERVAL if you want to use interval sealing.
+    # Other options you may want to add depending on your test scenario.
+    environment:
+      - SEALING_MODE=interval
+      - SEALING_INTERVAL=1
+    #   - CREATE_EMPTY_BLOCKS=true
+    # The 'command' may contain additional CLI options to the Frequency node,
+    # such as:
+    # --state-pruning=archive
+    command: --offchain-worker=always --enable-offchain-indexing=true
+    ports:
+      - 9944:9944
+    networks:
+      - social-app-backend
+    volumes:
+      - chainstorage:/data
+
+  ipfs:
+    image: ipfs/kubo:latest
+    ports:
+      - 4001:4001
+      - 5001:5001
+      - 8080:8080
+    networks:
+      - social-app-backend
+    volumes:
+      - ipfs_data:/data/ipfs
+
+  content-publishing-service-api:
+    image: amplicalabs/content-publishing-service:latest
+    platform: linux/amd64
+    ports:
+      - 3010:3000
+    environment:
+      <<: [*common-environment, *content-publishing-env]
+    depends_on:
+      - redis
+      - frequency
+      - ipfs
+    networks:
+      - social-app-backend
+
+  content-publishing-service-worker:
+    image: amplicalabs/content-publishing-service:latest
+    platform: linux/amd64
+    environment:
+      <<: [*common-environment, *content-publishing-env]
+      START_PROCESS: worker
+    depends_on:
+      - redis
+      - frequency
+      - ipfs
+    networks:
+      - social-app-backend
+
+  content-watcher-service:
+    image: amplicalabs/content-watcher-service:latest
+    platform: linux/amd64
+    ports:
+      - 3011:3000
+    environment:
+      <<: [*common-environment, *content-watcher-env]
+    depends_on:
+      - redis
+      - frequency
+      - ipfs
+    networks:
+      - social-app-backend
+
+  graph-service-api:
+    image: amplicalabs/graph-service:latest
+    platform: linux/amd64
+    ports:
+      - 3012:3000
+    environment:
+      <<: [*common-environment, *graph-service-env]
+      START_PROCESS: api
```

Comment:
Would like to check if graph-service can use the `command:` like account-service